lib/tasks/one_off.rake

```diff
@@ -103,4 +103,31 @@ namespace :one_off do
       end
     puts "Catalog page idx #{idx} (#{(idx / catalog_size.to_f) * 100}% complete)"
   end
+
+  # @param start_id String The id of the Project to resume from,
+  #                         to skip already-processed projects (default: nil)
+  # @param commit String Must equal 'yes' to perform changes
+  desc "Backfill maven dependencies after bugfixes"
+  task :backfill_maven_dependencies, %i[start_id commit] => :environment do |_t, args|
+    start = args.start_id
+    commit = args.commit == "yes"
+
+    bugfix_timestamp = Date.new(2024, 8, 28).beginning_of_day
+
+    projects = Project.platform("Maven")
+      .not_removed
+      .joins(:repository_maintenance_stats)
+      .where("last_synced_at < ?", bugfix_timestamp)
```

Comment:
I get an ambiguous column error when I try and run this query.

```suggestion
      .where("projects.last_synced_at < ?", bugfix_timestamp)
```

Reply:
ope just fixed that